File: webservice/Frikollection_Api/Frikollection_Api/DTOs/Collection/CollectionDto.cs
namespace Frikollection_Api.DTOs.Collection
{
    public class CollectionDto
    {
        public Guid CollectionId { get; set; }
        public string Name { get; set; }
        public bool Private { get; set; }
        public DateOnly CreationDate { get; set; }
        public int TotalProducts { get; set; }
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/collection/CollectionDto.kt
package org.example.frikollection_mobile_desktop.models.collection

import kotlinx.serialization.Serializable

@Serializable
data class CollectionDto(
    val collectionId: String,
    val name: String,
    val private: Boolean,
    val creationDate: String,
    val totalProducts: Int
)